src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -57,12 +57,7 @@ public TimeSpan AutomaticRefreshInterval
         }
 
         /// <summary>
-        /// The most recently retrieved configuration.
-        /// </summary>
-        internal BaseConfiguration CurrentConfiguration { get; set; }
-
-        /// <summary>
-        /// 12 hours is the default time interval that afterwards, <see cref="GetBaseConfigurationAsync(CancellationToken)"/> will obtain new configuration.
+        /// 12 hours is the default time interval that afterwards will obtain new configuration.
```

Comment:
With the changes, why do we still need the BaseConfigurationManager class? is there a reason the LastKnownGoodConfiguration can not be in the ConfigurationManager<BaseConfiguration>?